src/core/core.js

```diff
@@ -1041,6 +1041,72 @@ Crafty.fn = Crafty.prototype = {
             this._unbindAll();
             delete entities[this[0]];
         });
+    },
+
+    /**@
+     * #.freeze
+     * @comp Crafty Core
+     * @kind Method
+     * 
+     * @sign public this .freeze()
+     * 
+     * @triggers Freeze - While the entity is being frozen
+     * 
+     * Freezes the entity.  A frozen entity will not receive events or be displayed by graphics systems. 
+     * It is also removed from the spatial map, which means it will not be found by collisions, 
+     * raycasting, or similar functions.
+     * 
+     * Because the entity no longer listens to events, modifying its properties can result in an inconsistent state.
+     * 
+     * If custom components need to handle frozen entities, they can listen to the "Freeze" event, which will be triggered before the event system is disabled.
+     * 
+     * @see .unfreeze
+     */
+    freeze: function () {
```

Comment:
add the documentation about the event with `@trigger`
describe that freeze can be called on a group of entities with selector
add example for freezing group of entities